### PR TITLE
Port DA page

### PR DESCRIPTION
**Pull request by sdlyy**
Resolves L2B-5916

### CODE REVIEW

File: packages/frontend2/src/app/(new)/(other)/scaling/data-availability/_components/table/columns.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  TypeCell,
+  TypeColumnTooltip,
+} from '~/app/_components/table/cells/type-cell'
+import { sortSentiments } from '~/app/_components/table/sorting/functions/sentiment-sorting'
+import { type ScalingDataAvailabilityEntry } from '~/server/features/scaling/types'
+
+const columnHelper = createColumnHelper<ScalingDataAvailabilityEntry>()
+
+export const columns = [
+  // TODO: Make sure this is centered
```
Comment (torztomasz) [inline packages/frontend2/src/app/(new)/(other)/scaling/data-availability/_components/table/columns.tsx:16]: TODO

File: packages/frontend2/src/server/features/scaling/types.ts [hunk 2]
```diff
@@ -54,6 +55,28 @@ export interface ScalingSummaryLayer3sEntry {
...
+  purposes: ScalingProjectPurpose[]
+  stage: StageConfig | undefined
+  dataAvailability: DataAvailability
+  redWarning: string | undefined
+  showProjectUnderReview: boolean
+  hasImplementationChanged: boolean
+}
+
+type DataAvailability = {
+  layer: ValueWithSentiment<string>
```
Comment (torztomasz) [inline packages/frontend2/src/server/features/scaling/types.ts:75]: Do we want not typed sentiment here?

**Review by torztomasz — CHANGES_REQUESTED** [review]

File: packages/frontend2/src/server/features/scaling/types.ts (hunk 2 above)
Comment (sdlyy) [reply packages/frontend2/src/server/features/scaling/types.ts:75]: Could you elaborate?

File: packages/frontend2/src/app/(new)/(other)/scaling/data-availability/_components/table/columns.tsx (hunk 1 above)
Comment (sdlyy) [reply packages/frontend2/src/app/(new)/(other)/scaling/data-availability/_components/table/columns.tsx:16]: I'll do, but this is weird - I guess I'm not styling magician coz some of the index columns that are part of different tables have no styling and row denominator is perfectly centered

File: packages/frontend2/src/server/features/scaling/types.ts (hunk 2 above)
Comment (torztomasz) [reply packages/frontend2/src/server/features/scaling/types.ts:76]: Nevermind then

**Review by torztomasz — APPROVED** [review]

**Pull request merged**